.github/scripts/generate_changelog.py

```
@@ -89,7 +100,7 @@ def get_milestone_number(repo: str, title: str) -> int | None:
             "sort": "due_on",       # sort by due date
             "direction": "desc",    # most recently due first, so active milestones are found quickly
         }
-        resp = requests.get(url, headers=HEADERS, params=params, timeout=30)
+        resp = requests.get(url, headers=HEADERS, params=params)
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)